tests/fixtures/body-parameter/expected/Resource/TestResource.php

```diff
@@ -17,7 +17,7 @@ class TestResource extends Resource
     public function testSimpleBodyParameter($testString, $parameters = [], $fetch = self::FETCH_OBJECT)
     {
         $queryParam = new QueryParam();
-        $url        = '/test-simple';
+        $url        = 'http://localhost/test-simple';
         $url        = $url . ('?' . $queryParam->buildQueryString($parameters));
         $headers    = array_merge(['Host' => 'localhost'], $queryParam->buildHeaders($parameters));
```

Comment:
Yes probably. But my only concerns in the current behaviour is that by default it only works with the socket client and I think it can give a bad image of the library when using another http client =/